web/src/compose_reply.js

```diff
@@ -27,7 +27,10 @@ export function respond_to_message(opts) {
         if (message === undefined) {
             // Open empty compose with nothing pre-filled since
             // user is not focused on any table row.
-            compose_actions.start("stream", {trigger: "recent_view_nofocus"});
+            compose_actions.start("stream", {
+                trigger: "recent_view_nofocus",
+                keep_composebox_empty: opts.keep_composebox_empty,
```

Comment:
Just to confirm I understand correctly: This option is a noop in this case, right, because there won't be specified stream/topic, and thus `drafts.get_last_draft_based_on_compose_state` should return nothing? 

I think it's probably correct to have the option anyway.

Reply:
Yeah, that sounds right to me. To be clear, I'm defaulting in this function to just always pass through `opts.keep_composebox_empty` to `compose_actions.start` and didn't think about each case super throroughly, since I trust the caller to this function to make the decision. I'm open to being more stingy/cautious about it if you prefer that though.

Reply:
Yeah, I think the thing I'd wonder about is whether we should be passing through some subset of `opts` more generically, but I'm not sure I want to spend time on fiddling with that.